release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -211,6 +211,12 @@ To learn more, refer to the respective documentation for your cluster network pr
 If you use the OpenShift SDN cluster network provider, you can now use egress rules in network policy with `ipBlock` and `ipBlock.except`. You define egress policies in the `egress` array of the `NetworkPolicy` object.
 
 For more information, refer to xref:../networking/network_policy/about-network-policy.adoc#about-network-policy[About network policy].
+=======
```

Comment:
This line looks like it might have been left over from a merge conflict resolution? It's throwing off the rest of the text.